src/functionNameRule.ts

```diff
@@ -11,6 +11,36 @@ const PROTECTED_METHOD_REGEX = 'protected-method-regex';
 const STATIC_METHOD_REGEX = 'static-method-regex';
 const FUNCTION_REGEX = 'function-regex';
 
+const VALIDATE_PRIVATE_STATICS_AS_PRIVATE = 'validate-private-statics-as-private';
+const VALIDATE_PRIVATE_STATICS_AS_STATIC = 'validate-private-statics-as-static';
+const VALIDATE_PRIVATE_STATICS_AS_EITHER = 'validate-private-statics-as-either';
+const NOT_VALID_ARG = 'none';
```

Comment:
Pretending there isn't a history of discussion in the issue, I'm getting a little confused by these options. If someone just passes `[true]`, what are private statics validated as? How is that different from explicitly passing the `-either` option?

Reply:
Oh didn't know, got that updated!